### DISCUSSION TITLE
Fail when try to run uvicorn locally

### DISCUSSION THREAD

**Discussion by gabrielmrts** [Kludex/uvicorn#2082] [Q&A] [answered]
I'm trying to run uvicorn locally just for studies, so i cloned the repository, and tried to run the main.py file passing a test application, and received this error from cli:

class ColourizedFormatter(logging.Formatter):
AttributeError: partially initialized module 'logging' has no attribute 'Formatter' (most likely due to a circular import)

**Comment by Kludex** [comment, marked as answer]
You have the package on your PYTHONPATH i.e. you are running it from inside uvicorn/uvicorn, which is not possible.

**Reply by gabrielmrts** [reply]
Thanks, solved!
